lib/config/common.ts

```diff
@@ -80,7 +80,7 @@ export interface RenovateAdminConfig {
   dockerUser?: string;
 
   dryRun?: boolean;
-
+  secrets?: any;
```

Comment:
Is `Record<string, string>` a better fit here?

Reply:
Oh. 🙈 Sure should be a better type.

Reply:
I'm trying to remember if I needed to set this because it was pre-validation. Will see if defining it causes any TSC errors